### PR TITLE
 MTN use benchopt 1.5 API

### PR DESCRIPTION
**Pull request by wassimmazouz**

### CODE REVIEW

**Comment by wassimmazouz** [comment]
@mathurinm benchmark is up to date with `1.5` API

**Comment by mathurinm** [comment]
(closed/reopened to trigger the tests, after reenabling the workflow that had been disabled due to inactivity on the repo)

File: solvers/ChambollePockPDSplit.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def run(self, callback):
         sigma_w = 1.0 / (self.ratio * LA)
         # Init variables
         n, m = self.y.shape
-        u = np.zeros((n, m))
+        self.u = u = np.zeros((n, m))
```
Comment (mathurinm) [inline solvers/ChambollePockPDSplit.py]: sure?
I'm not very familiar with chaining =, if it can be avoided it's better IMO

Comment (wassimmazouz) [reply solvers/ChambollePockPDSplit.py]: `self.u` and `u` both need to be initialized for the solver to run correctly. [This syntax](https://github.com/wassimmazouz/benchmark_tv_1d/blob/6655e8bf3a48b7dab25b9e8eb77c2792e2d19a27/solvers/ADMM.py#L47) was used in `benchmark_tv_1d`.

I will make necessary changes then

**Comment by mathurinm** [comment]
Thanks heaps @wassimmazouz !

File: objective.py [hunk 2]
```diff
@@ -8,27 +8,21 @@
...
-    min_benchopt_version = "1.3"
+    min_benchopt_version = "1.5"
     name = "TV2D"
 
-    parameters = {'reg': [0.02],
-                  'delta': [0.9],
-                  'isotropy': ["anisotropic", "isotropic"],
-                  'data_fit': ["lsq", "huber"]}
-
-    def __init__(self, reg=0.02, delta=0.1,
```
Comment (mathurinm) [inline objective.py:19]: @wassimmazouz was there a particular reason to remove the init ? this slipped through

Comment (tomMoral) [reply objective.py:19]: there is no need for `__init__` to pass the parameters as we do this directly with `benchopt`, so removing it avoids having duplicated code, that you need to keep up to date when adding new parameters.

**Pull request merged**